rising/loading/hdf5.py

```diff
@@ -0,0 +1,148 @@
+from rising.loading.dataset import Dataset
+import os
+import numpy as np
+from tqdm import tqdm
+
+from rising.loading.collate import numpy_collate
+
+
+def h5ify(dset: Dataset, hdf5_path: str, dset_name: str, overwrite: bool = False,
```

Comment:
Nit: Can we make this function shorter or extract some general part into a second function? It is a bit long for my liking ^^

Reply:
I wouldn't split this, because there is no need to. We most likely don't need this code anywhere else...